tests/Functional/Controller/Api/PackageControllerTest.php

```diff
@@ -271,15 +271,16 @@ public function testUpdatePackage(): void
         $this->client->request('PUT', $this->urlTo('api_package_update', [
             'organization' => self::$organization,
             'package' => $packageId,
+        ]), [], [], [], (string) json_encode([
+            'url' => 'new-url',
+            'keepLastReleases' => 6,
         ]));
 
+        $package = $this->container()->get(DbalPackageQuery::class)->getById($packageId)->get();
+
         self::assertEquals(Response::HTTP_OK, $this->client->getResponse()->getStatusCode());
-        self::assertFalse(
-            $this->container()
-                ->get(DbalPackageQuery::class)
-                ->getById($packageId)
-                ->isEmpty()
-        );
+        self::assertEquals($package->url(), 'new-url');
+        self::assertEquals($package->keepLastReleases(), 6);
```

Comment:
:thinking: maybe this should be assertion for get package api request, instead of gutting it out of the database

Reply:
This is test for the `PUT` method, its outcome is modified record in the database. `GET` method has separate test